plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -84,6 +85,11 @@ export const createPublishGitlabMergeRequestAction = (options: {
             type: 'string',
             description: 'The token to use for authorization to GitLab',
           },
+          removeSourceBranch: {
+            title: 'Delete source branch',
+            type: 'string',
```

Comment:
This should probably be type `boolean` instead, as that's what the GitLab API wants (and it would look nicer in the ui): https://github.com/jdalrymple/gitbeaker/blob/master/packages/core/src/resources/MergeRequests.ts#L36

Reply:
Updated to boolean as per latest commit. Thank you.